File: 01_Basico/03 - Lacos_Repeticao/Respsota_em_Python/questao22.py
while True:
    maior_salto = -100
    menor_salto = 100
    nome = input("Digite seu nome: ")
    salto1 = float(input("Digite a altura do salto: "))
    salto2 = float(input("Digite a altura do salto: "))
    salto3 = float(input("Digite a altura do salto :"))
    salto4 = float(input("Digite a altura do salto: "))
    salto5 = float(input("Digite a altura do salto: "))
    
    if salto1 > maior_salto:
        maior_salto = salto1
    elif salto1 < menor_salto:
        menor_salto = salto1
    
    if salto2 > maior_salto:
        maior_salto = salto2
    elif salto2 < menor_salto:
        menor_salto = salto2
    
    if salto3 > maior_salto:
        maior_salto = salto3
    elif salto1 < menor_salto:
        menor_salto = salto3
    
    if salto4 > maior_salto:
        maior_salto = salto4
    elif salto4 < menor_salto:
        menor_salto = salto4
    
    if salto5 > maior_salto:
        maior_salto = salto1
    elif salto5 < menor_salto:
        menor_salto = salto5
    
    media = (salto1 + salto2 + salto3 + salto4 + salto5 - maior_salto - menor_salto) / 5
    break

print("")
print(f"\nAtleta: {nome}")
print(f"Primeiro salto: {salto1:2}m")
print(f"Segundo salto: {salto2:2}m")
print(f"Terceiro salto: {salto3:2}m")
print(f"Quarto salto: {salto4:2}m")
print(f"Quinto salto: {salto5:2}m")
print(f"\nMelhor salto: {maior_salto}m")
print(f"Pior salto: {menor_salto}m")
print(f"Média dos demais saltos: {media}m ")
print(f"\nResultado final\n{nome}: {media}")

File: 01_Basico/03 - Lacos_Repeticao/Resposta_em_Java_Script/questao22.js
while (true){
    var nome = prompt("Digite seu nome: ")
    var salto1 = parseFloat(prompt("Digite a altura do salto: "))
    var salto2 = parseFloat(prompt("Digite a altura do salto: "))
    var salto3 = parseFloat(prompt("Digite a altura do salto :"))
    var salto4 = parseFloat(prompt("Digite a altura do salto: "))
    var salto5 = parseFloat(prompt("Digite a altura do salto: "))
    
    var maior_salto = salto1
    var menor_salto = salto1
    
    if (salto2 > maior_salto) {
        maior_salto = salto2
    }else if (salto2 < menor_salto) {
        menor_salto = salto2
    }

    if (salto3 > maior_salto) {
        maior_salto = salto3
    }else if (salto3 < menor_salto) {
        menor_salto = salto3
    }
    
    if (salto4 > maior_salto) {
        maior_salto = salto4
    }else if (salto4 < menor_salto) {
        menor_salto = salto4
    }

    if (salto5 > maior_salto) {
        maior_salto = salto5
    }else if (salto5 < menor_salto) {
        menor_salto = salto5
    }
    
    media = (salto1 + salto2 + salto3 + salto4 + salto5 - maior_salto - menor_salto) / 5
    break
}

console.log(`\nAtleta: ${nome}`)
console.log(`Primeiro salto:${salto1}m`)
console.log(`Segundo salto:${salto2}m`)
console.log(`Terceiro salto:${salto3}m`)
console.log(`Quarto salto:${salto4}m`)
console.log(`Quinto salto:${salto5}m`)
console.log(`\nMelhor salto: ${maior_salto}m`)
console.log(`Pior salto: ${menor_salto}m`)
console.log(`Média dos demais saltos: ${media}m`)
console.log(`\nResultado final\n${nome}: ${media}m`)